src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -125,4 +125,11 @@ public void clear() {
         return internalList.iterator();
     }
 
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof UniquePersonList // instanceof handles nulls
+                && this.internalList.equals(
```

Comment:
`||` and `&&` should not be aligned if they are not at the 'same level' in the condition tree. In this case, `||` is at a higher branch of the condition tree than `&&`.